docs/src/development/variables/set-variables.md

```diff
@@ -23,17 +23,42 @@ Application variables are available at both build time and runtime.
 ## Create project variables
 
 Add secrets for all environments in project variables
-using [the Console](../../administration/web/configure-project.md#variables) or the CLI.
+using [the Console](../../administration/web/configure-project.md#variables) or the [CLI](../../administration/cli/_index.md).
 
-For example, to create the project variable `foo` with the value `bar`, run:
+For example, you may need to set a variable to vary between production and other environments, such as API gateway credentials.
+You can set the non-production version as a project variable
+and then set a [variable specific to the production environment](#create-environment-specific-variables) to override it.
```

Comment:
🛠️ I'm not sure I understand this correctly. What I understand is something along the lines of this:

```suggestion
For example, you may need to use API gateway credentials for your production environment that are different from those used for your non-production environments. 
To do so, you can set the non-production credentials as a project variable
and set another [variable specific to your production environment](#create-environment-specific-variables) to override it.
```
Maybe I still don't get it, and the phrasing still probably isn't perfect, but my point is that I think clarity could be improved here.

Reply:
This was wording that I didn't change as part of this PR. I can update it a little, but I didn't want to get into the details here too much. I think `another` is a little confusing, because the point was that there are two types of variables and `another` makes them seem basically the same. But can do at least something.